### DISCUSSION TITLE
[Bug]: Portal Log-in with Firefox not possible

### DISCUSSION THREAD

**Discussion by cami0092** [pimcore/pimcore#13914] [Q&A / Support]
### Expected behavior

The log-in should work and take me to the portal sites.

### Actual behavior

After clicking the log-in button the log-in pages refreshes and deletes my input.

### Steps to reproduce

Put in your log-in credentials into the input field and click "Login"

**Comment by mattamon** [comment]
Hey @cami0092,

I just tested it on our demo version and everything works fine.
I am currently on version 108.0.1 of firefox.

Which version are you using? 
Does the login work in other browsers?

**Comment by cami0092** [comment]
Hey Matthias,

I’m also using the 108.0.1 (64-bit) Firefox Version. And yes it works fine with Safari, Chrome and Edge. Just Firefox is making problems but only since a couple of days. It definitely used to work. Also it can’t be my computer because everyone in our team can not log in through firefox.

Best regards,
Camilla Piontek

Am 23.12.2022 um 11:20 schrieb Matthias S. ***@***.***>:



Hey @cami0092<https://github.com/cami0092>,

I just tested it on our demo version and everything works fine.
I am currently on version 108.0.1 of firefox.

Which version are you using?
Does the login work in other browsers?

**Comment by mattamon** [comment]
Hey Camilla,

ah okay thanks for the info! Can you please in the meanwhile use any other browser than firefox.
Since I cannot replicate this problem, I would need some more info. 
Like can you can in the network tab for any error messages that maybe occur. 

Maybe is has something to do with your IT, since this problem just occued a couple of days ago?
